plugin/arcconf/arcconf.py

```diff
@@ -0,0 +1,919 @@
+                        temp_location = temp_list[1].split(',')
+                        if temp_list[1].find('Connector Not Applicable') > 0:
+                            physical_data.update({temp_list[0] : temp_location[0]})
+                            continue
+                        else:
+                            physical_data.update({temp_list[0] : temp_location[0]})
+                            physical_data.update({'Slot' : temp_location[1].split('(')[0]})
+                            physical_data.update({'Connector' : temp_location[1].split('(')[1]})
+                            check_line = output_lines[line_num - 1]
+                            check_line = check_line.split(":")
+                            channel_device = check_line[2].split(",")
+                            channel = channel_device[0]
+                            device = channel_device[1].split("(")[0]
+                            physical_data.update({'Channel':channel})
+                            physical_data.update({'Device':device})
+                    elif temp_list[0] == 'Model':
+                        check_line = output_lines[line_num - 2]
+                        if check_line.find("Type") > 0:
+                            continue
+                        else:
+                            physical_data.update({temp_list[0] : temp_list[1]})
+                    else:
+                        physical_data.update({temp_list[0] : temp_list[1]})
+                    if temp_list[0] == 'SSD':
+                        list_data.append(physical_data.copy())
+                        physical_data = {}
+                else:
+                    continue
+
+        if component == 'Logical Device':
+            for info_section in getconfig_logical_info_section:
+                if info_section == temp_list[0]:
+                    logical_data.update({temp_list[0] : temp_list[1]})
+                elif temp_list[0].find('Segment ') != -1:
+                    serialNo = temp_list[1].split(')')[1]
+                    serialNo = serialNo.strip()
+                    logical_data.update({'Serial number' : serialNo})
+                    list_data.append(logical_data.copy())
+                    break
+                else:
+                    continue    
+
+    if component == "Adapter":
+        return cntrl_data
+    return list_data
+
+
+def _arcconf_size_to_lsm(arcconf_size):
+    """
+    ARCCONF displays the disk size in terms of  'TB, GB, MB, KB'
+    for LSM, they are 'TiB' and etc.
+    Returning int of block bytes
+    """
+    re_regex = re.compile("^([0-9.]+) +([EPTGMK])B")
+    re_match = re_regex.match(arcconf_size)
+    if re_match:
+        return size_human_2_size_bytes(
+            "%s%siB" % (re_match.group(1), re_match.group(2)))
+
+    raise LsmError(
+        ErrorNumber.PLUGIN_BUG,
+        "_arcconf_size_to_lsm(): Got unexpected size string %s" %
+        arcconf_size)
+
+def _lsm_size_to_arcconf_MB(lsm_size):
```

Comment:
How about `_lsm_size_bytes_to_arcconf_mb()`? The `lsm_size` means string like `1.0 GiB`.